hazelcast/src/main/java/com/hazelcast/config/alto/AltoSocketConfig.java

```diff
@@ -0,0 +1,63 @@
+package com.hazelcast.config.alto;
+
+import com.hazelcast.config.InvalidConfigurationException;
+
+import javax.annotation.Nonnull;
+import java.util.regex.Pattern;
+
+public class AltoSocketConfig {
+    private static final String DEFAULT_PORT_RANGE = "11000-21000";
+    private static final int DEFAULT_RECEIVE_BUFFER_SIZE = 128 * 1024;
+    private static final int DEFAULT_SEND_BUFFER_SIZE = 128 * 1024;
+
+    private String portRange = DEFAULT_PORT_RANGE;
+    private int receiveBufferSize = DEFAULT_RECEIVE_BUFFER_SIZE;
+    private int sendBufferSize = DEFAULT_SEND_BUFFER_SIZE;
+
+    public String getPortRange() {
+        return portRange;
+    }
+
+    public AltoSocketConfig setPortRange(@Nonnull String portRange) {
+        if (!Bounds.PORT_RANGE_PATTERN.matcher(portRange).matches()) {
+            throw new InvalidConfigurationException("Invalid port definition");
+        }
+
+        this.portRange = portRange;
+        return this;
+    }
+
+    public int getReceiveBufferSize() {
+        return receiveBufferSize;
+    }
+
+    public AltoSocketConfig setReceiveBufferSize(int receiveBufferSize) {
+        if (receiveBufferSize < Bounds.MIN_BUFFER_SIZE || receiveBufferSize > Bounds.MAX_BUFFER_SIZE) {
```

Comment:
I'm not sure if the reference to 'Bounds' is a desirable design. It increases coupling.

Reply:
What do you prefer then?

- Move static fields to config class
- Remove fields and use integers directly
- Move them to a separate class

Reply:
The Bounds class is causing coupling, it is a fundamentally problematic design. 

So just add it to the AltoSocketConfig as constant.